src/System.Diagnostics.TraceSource/src/Properties/InternalsVisibleTo.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//------------------------------------------------------------
+
+//------------------------------------------------------------
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("System.Diagnostics.TraceSource.Tests, PublicKey=002400000480000094000000060200000024000052534131000400000100010007d1fa57c4aed9f0a32e84aa0faefd0de9e8fd6aec8f87fb03766c834c99921eb23be79ad9d5dcc1dd9ad236132102900b723cf980957fc4e177108fc607774f29e8320e92ea05ece4e821c0a5efe8f1645c4c0c93c1ab99285d622caa652c1dfad63d745d6f2de5f17e5eaf0fc4963d261c8a12436518206dc093344d5ad293")]
```

Comment:
The two classes that are impacted are TraceInternal and TraceListenerCollection. TraceInternal can be mostly tested through Trace. TraceListenerCollection has an internal constructor, so options would be to either allow internals, not test it, or use reflection to create instances of it. Would using reflection to construct it be preferred to internals visible to?